.github/workflows/test.yml

```diff
@@ -6,6 +6,7 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v2
-
       - name: Build project
         run: ./gradlew clean build
+      - name: Build Docker image
+        run: docker build . -t paymenthubee.mifos.io/phee/notifications
```

Comment:
Is this tag pointing to an active container registry? Or is it just a name/placeholder?

Reply:
Indeed it's just a placeholder - let me know if I should use something else

Reply:
Should I change those or are they good to go?